resources/conf/wdio-bstack-local-parallel-devices.conf.js

```diff
@@ -0,0 +1,101 @@
+var defaults = require("./wdio.conf.js");
+var _ = require("lodash");
+
+var overrides = {
+  onBrowserstack: true,
+
+  user: process.env.BROWSERSTACK_USER,
+  key: process.env.BROWSERSTACK_ACCESSKEY,
+
+  specs: ["./test/specs/local/local.spec.js"],
+
+  capabilities: [
```

Comment:
can we use commonCapabilities for values of build, project etc and all other caps which are truly common for the test run?

an example we had created for a customer:

https://github.com/gregjopa/webdriver-io-boilerplate/blob/main/resources/config/wdio.prod.browserstack.conf.ts#L16

Reply:
implemented similar commonCapabilities